### PR TITLE
RHDEVDOCS-4932 - Logging 5.5.9 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.10+ 

Issue: [RHDEVDOCS-4932](https://issues.redhat.com//browse/RHDEVDOCS-4932)

Link to docs preview: https://57318--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-5-9_cluster-logging-release-notes-v5x 

QE review:
- [x] QE has approved this change.

Additional information: **Only cluster-logging-rn-5.5.9.adoc requires review**. Other files are duplicates of live content for sub-folder population.

### CODE REVIEW

File: modules/logging-rn-5.5.6.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+* Before this update, the reconciliation of the `ClusterLogForwarder` custom resource would incorrectly report a degraded status of one or more pipelines causing the collector pods to restart every 8-10 seconds. With this update, reconciliation of the `ClusterLogForwarder` custom resource processes correctly, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3469[LOG-3469])
+
+* Before this change the spec for the `outputDefaults` field of the ClusterLogForwarder custom resource would apply the settings to every declared Elasticsearch output type. This change corrects the behavior to match the enhancement specification where the setting specifically applies to the default managed Elasticsearch store. (link:https://issues.redhat.com/browse/LOG-3342[LOG-3342])
+
+* Before this update, the the OpenShift CLI (oc) `must-gather` script did not complete because the OpenShift CLI (oc) needs a folder with write permission to build its cache. With this update, the OpenShift CLI (oc) has write permissions to a folder, and the `must-gather` script completes successfully. (link:https://issues.redhat.com/browse/LOG-3472[LOG-3472])
```
Comment (mramendi) [inline modules/logging-rn-5.5.6.adoc:25]: oc is a command and so should be monospace, I think

File: modules/logging-rn-5.5.6.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+* Before this update, the Loki Operator webhook server caused TLS errors. With this update, the Loki Operator webhook PKI is managed by the Operator Lifecycle Manager's dynamic webhook management resolving the issue. (link:https://issues.redhat.com/browse/LOG-3511[LOG-3511])
```
Comment (mramendi) [inline modules/logging-rn-5.5.6.adoc:27]: I would put a comma after "management"

File: modules/logging-rn-5.5.7.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
+//module included in logging-5-5-release-notes.adoc
+:_content-type: REFERENCE
+[id="logging-release-notes-5-5-7_{context}"]
+= Logging 5.5.7
+This release includes link:https://access.redhat.com/errata/RHSA-2023:0633[OpenShift Logging Bug Fix Release 5.5.7].
+
+[id="logging-5-5-7-bug-fixes"]
+== Bug fixes
+* Before this update, the LokiStack Gateway Labels Enforcer generated parsing errors for valid LogQL queries when using combined label filters with boolean expressions. With this update, the LokiStack LogQL implementation supports label filters with boolean expression and resolves the issue. (link:https://issues.redhat.com/browse/LOG-3534[LOG-3534])
```
Comment (mramendi) [inline modules/logging-rn-5.5.7.adoc:9]: Grammar requires either "filters with a boolean expression" or "filters with boolean expressions".

File: modules/logging-rn-5.5.7.adoc [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+* Before this update, the `ClusterLogForwarder` custom resource (CR) did not pass TLS credentials for syslog output to Fluentd, resulting in errors during forwarding. With this update, credentials pass correctly to Fluentd, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3533[LOG-3533])
```
Comment (mramendi) [inline modules/logging-rn-5.5.7.adoc:11]: "for syslog output to Fluent" reads like syslog output goes fo Fluentd, while actually it's credentials. But I am not sure how to fix this.

File: modules/logging-rn-5.5.9.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
+//module included in logging-5-5-release-notes.adoc
+:_content-type: REFERENCE
+[id="logging-release-notes-5-5-9_{context}"]
+= Logging 5.5.9
+This release includes link:https://access.redhat.com/errata/RHBA-2023:1310[OpenShift Logging Bug Fix Release 5.5.9].
+
+[id="logging-5-5-9-bug-fixes"]
+== Bug fixes
+* Before this update, a problem with the Fluentd collector caused it to not capture OAuth login events stored in /var/log/auth-server/audit.log. This led to incomplete collection of login events from the OAuth service. With this update, the Fluentd collector now resolves this issue by capturing all login events from the OAuth service, including those stored in /var/log/auth-server/audit.log, as expected.(link:https://issues.redhat.com/browse/LOG-3730[LOG-3730])
+
```
Comment (mramendi) [inline modules/logging-rn-5.5.9.adoc:10]: File (path)names should be monospace.

**Review by mramendi — COMMENTED** [review]
First round of peer review done, the issues found are small but somewhat numerous, I would prefer to have a second look after they are fixed, unless this is time-critical.

File: modules/logging-rn-5.5.6.adoc [hunk 6]
```diff
@@ -0,0 +1,49 @@
+//module included in logging-5-5-release-notes.adoc
+:_content-type: REFERENCE
+[id="logging-release-notes-5-5-6_{context}"]
+= Logging 5.5.6
+This release includes link:https://access.redhat.com/errata/RHBA-2023:0386[OpenShift Logging Bug Fix Release 5.5.6].
+
+[id="logging-5-5-6-bug-fixes"]
+== Bug fixes
+* Before this update, the Pod Security admission controller added the label `podSecurityLabelSync = true` to the `openshift-logging` namespace. This resulted in our specified security labels being overwritten, and as a result Collector pods would not start. With this update, the label `podSecurityLabelSync = false` preserves security labels. Collector pods deploy as expected. (link:https://issues.redhat.com/browse/LOG-3340[LOG-3340])
```
[the remark this answers is not in the document]
Comment (libander) [reply modules/logging-rn-5.5.6.adoc:9]: This is a duplicated file of release notes which are already live. The only file that includes new content requiring review is modules/cluster-logging-rn-5.5.9.adoc

**Comment by libander** [comment]
Note: Only cluster-logging-rn-5.5.9.adoc contains new content requiring review. Other files are duplicates of live content for sub-folder population.

**Review by mramendi — APPROVED** [review]

**Pull request merged**